terraform-sentinel/step1.md

```diff
@@ -0,0 +1,90 @@
+In this scenario, you will apply the Sentinel logic principles to a Terraform specific deployment.
+
+Navigate to the `terraform-sentinel/tf-config/main.tf`{{open}} and review the configuration you are testing.
+
+This configuration builds an S3 bucket with a unique name and deploys an example web app as a bucket object. You have an S3 bucket policy attached to the bucket resource which allows pubic read permissions for your bucket object. This example configuration does not have any deployment safeguards built in and if your AWS user has S3 build and upload permissions, your Terraform deployment will apply successfully.
+
+Proper testing of a policy requires that these values be able to be mocked - or, in other words, simulated in a way that allows the accurate testing of the scenarios that a policy could reasonably pass or fail under.
+
+In order to test this policy, this scenario has pre-generated [mock data from Terraform Cloud](https://www.terraform.io/docs/cloud/sentinel/mock.html). To learn how to generate mock data for testing in the Sentinel CLI, follow the [Sentinel Learn guides on Sentinel & Terraform Cloud](https://www.learn.hashicorp.com/terraform?LINK).
```

Comment:
Let's not link out unless you think users will have extra time. Because these scenarios tend to time out we don't want users poking around too much.

Reply:
I think we can probably just say "Look at the files XYZ. These contain the mock data you will use to test your policy."